features/invite_user.feature

```diff
@@ -13,26 +13,10 @@ Feature: Invite user
     Then I see a user created message
     And the email address is in the list of users
 
```

Comment:
I think it would be good to get these tests - and additional tests passing - as this was initially a spike I guess you plan to update and add in the next trello task.

Reply:
Yep, didn't want to write lots of tests when I wasn't sure of the implementation. Was going to add those when I implement all of the roles